index.rst

```diff
@@ -4,54 +4,146 @@
-  _static/ directory. The reST syntax for adding the image is
-
-  .. figure:: /_static/filename.ext
-     :name: fig-label
-
-     Caption text.
-
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
-
-   Feel free to delete this instructional comment.
 
 :tocdepth: 1
 
 .. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
 .. sectnum::
 
-.. TODO: Delete the note below before merging new content to the master branch.
 
-.. note::
 
-   **This technote is not yet published.**
+Introduction
+============
+This document describes how to use the Curvature WaveFront Sensing algorithm for optical collimation during
+operation of the Auxiliary Telescope.
+
+The Auxiliary Telescope is an F/18 Ritchey-Chretien with a 1.2 meter diameter near-spherical mirror.
+The primary mirror directs the light to a 218mm diameter convex secondary which directs the light to a tertiary (flat)
+fold mirror which rotates to direct the light into one of the two nasmyth ports. The primary mirror is supported by
+24 pneumatic actuators and 3 hardpoints, where the pressure is adjusted as a function of elevation (see technote X).
+The secondary mirror is mounted to a PI H-824 Hexapod which is primarily controlled from the Auxiliary Telescope
+Active Optics System (ATAOS) Controllable SAL Component (CSC). Although the hexapod has 6 degrees of freedom, the
+mechanical mounting is believed to be sufficiently accurate such that to first-order any tilts can be mapped as
+a decenter of the mirror. This means that after rough collimation (utilizing an alignment scope and laser),
+the alignment of the telescope consists of 4 free parameters: primary mirror pressure, hexapod X and Y decenter, and
+hexapod piston (defocus).
+
+For the purposes of this document, the important parameters are:
+
++------------------------------------+-------------------+
+| Effective Focal length             |  21.6 m           |
++------------------------------------+-------------------+
+| Magnification between secondary    |  41               |
+| motion and back-focal distance     |                   |
++------------------------------------+-------------------+
+| Pixel pitch                        |  10um             |
++------------------------------------+-------------------+
+| Plate Scale                        |  100 um/arcsec    |
+|                                    |  0.1 arcsec/pixel |
++------------------------------------+-------------------+
+| Secondary Obscuration Diameter     |  0.423m           |
++------------------------------------+-------------------+
+
+The optical design (in Zemax) for the telescope and LATISS instrument can be found in
+`Document-27702 <https://docushare.lsst.org/docushare/dsweb/Get/Document-27702>`_ (credentials required). The
```

Comment:
Double "The". One at the end of this line and one at the beginning of the next line.